iModelCore/libsrc/curl/BeCurl.mke

```diff
@@ -342,8 +342,6 @@ $(o)krb5_gssapi$(oext) : $(BeCurlLibDir)vauth/krb5_gssapi.c ${MultiCompileDepend
 
 $(o)krb5_sspi$(oext) : $(BeCurlLibDir)vauth/krb5_sspi.c ${MultiCompileDepends}
 
-$(o)ntlm$(oext) : $(BeCurlLibDir)vauth/ntlm.c ${MultiCompileDepends}

```

Comment:
unsure why this is here, but probably from merging main branch with libsrc-Main. @tcobbs-bentley, if builds succeed i guess its fine, but can you check this deletion should be here, or if I should revert.